brayns/common/PropertyMap.cpp

```diff
@@ -212,7 +212,7 @@ void PropertyMap::merge(const PropertyMap& input)
         };
     };
 
-    const auto isCompatbileTypes = [](Property::Type t0, Property::Type t1) {
+    const auto areCompatibleTypes = [](Property::Type t0, Property::Type t1) {
         return (t0 == Property::Type::Int && t1 == Property::Type::Double) ||
                (t0 == Property::Type::Double && t1 == Property::Type::Int) ||
```

Comment:
This conversion should not be allowed @karjonas, any reason for allowing it?

Reply:
When we get doubles from the UI they are sent as `0` instead of `0.0` so therefore we need to be able to convert between double and float.

Reply:
But that would be a type promotion, which is fine. The condition is also accepting type demotion (accepting double for an integer), which is what I don't see why should be necessary.

Reply:
Yes, should be fine, you just need to change the order in the jsonToPropertyMap method in jsonPropertyMap.h.